frontend/components/httpsUrlInput.js

```diff
@@ -13,7 +13,7 @@ const HttpsUrlInput = ({ placeholder, ...props }) => (
   <input
     {...props}
     type="url"
-    pattern="https://.+"
+    pattern="https://.+\.\w{2,}"
```

Comment:
That's probably a good idea, but should be a new issue IMHO. We'd also want to validate that on the back end, and probably make it explicit in the docs.